mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -106,6 +106,11 @@ class LogisticRegression(override val uid: String)
       case LabeledPoint(label: Double, features: Vector) => (label, features)
     }
     val handlePersistence = dataset.rdd.getStorageLevel == StorageLevel.NONE
+    trainOnInstances(instances, handlePersistence)
+  }
+
+  protected[spark] def trainOnInstances(instances: RDD[(Double, Vector)],
```

Comment:
We may just pass in `RDD[LabeledPoint]` from `mllib` to `ml` since the implicitly conversion. In `ALS`, the `mllib` is calling the new implementation in `ml` using the same way.
https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/recommendation/ALS.scala